spec/models/importers/hmis_six_one_one/base_spec.rb

```diff
@@ -234,6 +234,73 @@
     end
   end
 
+  describe 'When restoring geographies' do
+    let!(:data_source) { create :data_source_fixed_id }
+    let!(:fixed_date) { Date.parse('2019-01-01') }
+
+    let!(:project_1) { create :grda_warehouse_hud_project, ProjectID: 1 }
+    let!(:project_2) { create :grda_warehouse_hud_project, ProjectID: 2 }
+    let!(:geography) { create :grda_warehouse_hud_geography, GeographyID: 10, DateDeleted: fixed_date, ProjectID: project_2.ProjectID }
+
+    it 'has one deleted geography record' do
+      expect(GrdaWarehouse::Hud::Geography.only_deleted.count).to eq(1)
+    end
+
+    it 'has two projects' do
+      expect(GrdaWarehouse::Hud::Project.count).to eq(2)
+    end
+
+    describe 'with both projects' do
+      # This is a before instead of a before(:all) because we can't reference 'let's in a before(:all)
+      before do
+        @delete_later = []
+        file_path = 'spec/fixtures/files/importers/hmis_six_on_one/project_and_geography_restore_by_both_projects_files'
+        import(file_path, data_source)
+      end
+      after do
+        # Because we are only running the import once, we have to do our own DB and file cleanup
+        GrdaWarehouse::Utility.clear!
+        cleanup_files
+      end
+
+      it 'restores and updates the deleted geography record' do
+        expect(GrdaWarehouse::Hud::Geography.only_deleted.count).to eq(0)
+
+        geography.reload
+        expect(geography.CoCCode).to eq('KY-500')
+      end
+
+      it 'has two projects' do
+        expect(GrdaWarehouse::Hud::Project.count).to eq(2)
+      end
+    end
+
+    describe 'with project 1' do
+      # This is a before instead of a before(:all) because we can't reference 'let's in a before(:all)
+      before do
+        @delete_later = []
+        file_path = 'spec/fixtures/files/importers/hmis_six_on_one/project_and_geography_restore_by_one_project_files'
+        import(file_path, data_source)
+      end
+      after do
+        # Because we are only running the import once, we have to do our own DB and file cleanup
+        GrdaWarehouse::Utility.clear!
+        cleanup_files
+      end
+
+      it 'restores and updates the deleted geography record' do
+        expect(GrdaWarehouse::Hud::Geography.only_deleted.count).to eq(0)
+
+        geography.reload
+        expect(geography.CoCCode).to eq('KY-500')
+      end
+
+      it 'has two projects' do
+        expect(GrdaWarehouse::Hud::Project.count).to eq(2)
+      end
```

Comment:
Couple more tests. 
1. We should confirm that the restored geography belongs to the correct project.
2. We should test this if the updated at on the deleted record is newer than the incoming (I *think* this would not restore the record, but I could be wrong.)  For now, the test can just be documentation of the existing logic.
3. We should test this if the updated at on the deleted record is the same date as the incoming
4. Test to confirm the project that had the deleted geography no longer has any geography records.